tools/hooks/requirements_changed.rb

```diff
@@ -4,7 +4,7 @@
 # of these are currently used, but are included for future-proofing.
 REQUIREMENTS = {
   "requirements.txt" => "pip install -r requirements.txt",
-  "package.json" => "npm install",
+  "package.json" => "npm prune && npm update",
```

Comment:
not too sure exactly where this is used, but I think it's just for helpful text explaining to the user what they should do